tests/integration/test_cli.sh

```diff
@@ -58,3 +58,17 @@ test_get_commissioner_dataset()
 
     stop_daemon
 }
+
+test_cancel_command() {
+    set -e
+
+    start_commissioner "${NON_CCM_CONFIG}"
+
+    ## Expecting error CANCELLED after sending SIGINT
+    send_command_to_commissioner "borderagent discover" "CANCELLED" &
+    sleep 1
+
+    pkill -s SIGINT "${COMMISSIONER_CLI}"
```

Comment:
should stop gracefully and return 0?

Reply:
Oh, this is to send an interrupt signal (Ctrl+C) to the commissioner CLI to cancel requests. see the implementation:

https://github.com/openthread/ot-commissioner/blob/052958e7fc05eddad2666c42e7dd5e8494d7ff0b/src/app/cli/main.cpp#L80-L83

Added comments to make this clearer.